docs-src/concepts/what-is-a-workflow-execution.md

```diff
@@ -38,6 +38,16 @@ Scalability is responsiveness in the presence of load.
 A single Workflow Execution is limited in size and throughput but is scalable because it can [Continue-As-New](/concepts/what-is-continue-as-new) in response to load.
 A Temporal Application is scalable because the Temporal Platform is capable of supporting millions to billions of Workflow Executions executing concurrently, which is realized by the design and nature of the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) and [Worker Processes](/concepts/what-is-a-worker-process).
 
+### Replays
+
+A Replay is the method by which a Workflow Execution resumes making progress. During a Replay the Commands that are generated are checked against an existing Event History. Replays are necessary and often happen to give the effect that Workflow Executions are resumable, reliable, and durable.
+
+If a failure occurs, the Workflow Execution picks up where the last recorded event occurred in the Event History.
+
+To accomplish this, the Replay retries the Workflow Execution function and expects a response. If the corresponding response was recorded in the Event History, the Replay process continues to the next function. If a response wasn't recorded in the Event History, the Replay _retries_ that function and records the response in the Event History. This process repeats often.
+
+You can use Replay APIs from any of the Temporal SDKs to control this behavior.
```

Comment:
```suggestion
- [How to use Replay APIs to test Workflow Definitions](/app-dev-context/replays).
```

I think you can drop this explanation and focus on linking off to the "how to".

I think the way "response" is used to describe things here is too generic and so it doesn't explain much.
The "deterministic constraints" section does this anyways - and so linking to it above should be enough.